app/javascript/components/test_iterations_summary_table.jsx

```diff
@@ -1,17 +1,17 @@
 import React, { useState, useEffect } from "react";
-import actionCable from "actioncable";
+import "actioncable";
 
 import consumer from "../application/action_cable_consumer";
 
-function IterationSummary({ id, testsStatus }) {
+function TestIterationSummary({ id, testsStatus }) {
```

Comment:
Could we rename this to `Example...` rather than `Test...`?

Reply:
(or `Sample...`)

Reply:
Sure